packages/sirius-web/frontend/sirius-web/src/tools/PapayaOperationActivityLabelDetailToolContribution.tsx

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { NodeData, PaletteToolComponentProps } from '@eclipse-sirius/sirius-components-diagrams';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import { Slideshow } from '@material-ui/icons';
+import { Fragment, useState } from 'react';
+import { useNodes } from 'reactflow';
+
+const useToolStyle = makeStyles(() => ({
+  tool: {
+    width: '36px',
+  },
+}));
+
+type Modal = 'dialog';
+export const PapayaOperationActivityLabelDetailToolContribution = ({ diagramElementId }: PaletteToolComponentProps) => {
```

Comment:
I've not reviewed the PR yet but this code will have to be moved to `sirius-web-papaya` now that Papaya has a dedicated frontend.

Reply:
yes, totally, I had forgotten about this new split.